src/lib.rs

```diff
@@ -179,6 +179,14 @@ pub fn collect() {
 pub struct Debug;
 
 impl Debug {
+    /// Checks that the object was marked as reachable in the last collection.
```

Comment:
It's a bit difficult to know what true/false means here. How about saying `Returns `true` if this object was reachable in the last collection`?

Reply:
Sure